src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,30 +1,38 @@
-# TO-DO: Complete the selection_sort() function below 
-def selection_sort( arr ):
+# TO-DO: Complete the selection_sort() function below
+def selection_sort(arr):
     # loop through n-1 elements
-    for i in range(0, len(arr) - 1):
-        cur_index = i
-        smallest_index = cur_index
+    for i in range(0, len(arr)):
+        curr = i
+        smallest = curr
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
-
+        # (hint, can do in 3 loc)
+        # iterate over each item in the array; start at curr number, stop at len(arr)
+        for x in range(curr, len(arr)):
+            if arr[x] < arr[smallest]:
+                smallest = x
 
         # TO-DO: swap
-
-
-
+        # move to next number
+        temp = arr[smallest]
```

Comment:
Python has a cool syntax for this: `arr[smallest], arr[curr] = arr[curr], arr[smallest]`

Reply:
Refactored to cleaner syntax